apps/documentation/src/routes/documentation/contributing/pull-requests.mdx

```diff
@@ -24,10 +24,24 @@ you can check the [official documentation](https://docs.github.com/en/pull-reque
 
 ---
 
-- If you are fixing an issue, bear in mind to link the issue itself in the PR description.
-- If you are adding a new feature, describe the intended use case that the feature fulfills.
+- Please do:
 
-> Bug fixes and new features should include tests.
+  - Fill the template in full
+
+    - If you are fixing an issue, bear in mind to link the issue itself in the PR description.
+    - If you are adding a new feature, describe the intended use case that the feature fulfills.
+
+    > Bug fixes and new features should include tests.
```

Comment:
We don't have a good enough test suite to always ask for tests. Better asking team in case at each occurrence